task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -58,8 +61,9 @@ spec:
         image_with_digest="$(params.IMAGE_URL)@$(params.IMAGE_DIGEST)"
 
         if ! oc image extract "${image_with_digest}" ; then
-          echo "Unable to extract image! Skipping checking binaries!"
-          HACBS_TEST_OUTPUT="$(make_result_json -r ERROR -t 'Unable to extract image! Skipping checking binaries!')"
+          echo "Unable to extract image  so that it is unable to validate the extracted binaries of the image!"
```

Comment:
Nit: double space at `Unable to extract image  so`.
Suggestion: `Unable to extract image, unable to validate the extracted binaries of the image!`

Reply:
Fixed. Thanks.